2DArray (Easy).rb

```diff
@@ -1,12 +1,15 @@
-
+#There are 16  hourglasses in arr . An hourglass sum is the sum of an hourglass' values. Calculate the hourglass sum for every hourglass in , then print the maximum hourglass sum. The array will always be6 X 6.
 def hourglassSum(arr)
     # Write your code here
     temp = []
+    # Row Loop
     4.times do |i|
+        # Column Loop
         4.times do |j|
              temp << arr[i][j] + arr[i][j+1] + arr[i][j+2] + arr[i+1][j+1] + arr[i+2][j] + arr[i+2][j+1] + arr[i+2][j+2]
```

Comment:
This line might not be easily understandable for someone who doesn't know the problem well. Can we please move this to a method and give a good name for the method which describes what this is doing?